dotenv/src/iter.rs

```diff
@@ -21,7 +21,11 @@ impl<R: Read> Iter<R> {
         }
     }
 
-    /// Loads all variables found in the `reader` into the environment.
+    /// Loads all variables found in the `reader`, which not yet exist in the environment,
+    /// into the environment.
```

Comment:
```suggestion
    /// Loads all variables found in the `reader` into the environment, unless they already exist.
```
maybe this improves readability

Reply:
Agreed, thanks!